src/FilterResource.php

```diff
@@ -183,6 +183,26 @@ public function upperFirst()
     }
 
     /**
+     * Add camel-case which is a combo of Lower + UpperWords
+     *
+     * @return $this
+     */
+    public function camelCase()
+    {
+        return $this->lower()->UpperWords();
```

Comment:
If the function is a combination of filters, I don't think it should be added to the Filter API as a default rule. A programmer would be able to combine rules himself if he needs to.